src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -29,21 +29,22 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.openrewrite.java.testing.jmockit.JMockitBlockType.FullVerifications;
-import static org.openrewrite.java.testing.jmockit.JMockitBlockType.NonStrictExpectations;
+import static org.openrewrite.java.testing.jmockit.JMockitBlockType.*;
 
 class JMockitBlockRewriter {
 
     private static final String WHEN_TEMPLATE_PREFIX = "when(#{any()}).";
     private static final String VERIFY_TEMPLATE_PREFIX = "verify(#{any()}";
     private static final String VERIFY_NO_INTERACTIONS_TEMPLATE_PREFIX = "verifyNoMoreInteractions(";
+    private static final String VERIFY_IN_ORDER_TEMPLATE_PREFIX = "inOrder(";
+    //private static final String VERIFY_IN_ORDER_TEMPLATE_PREFIX = "InOrder inOrder = inOrder(";
```

Comment:
```suggestion
```

Reply:
Ah yes - this is one issue I'd like to get input on
inOrder(mock) works, but
InOrder inOrder = inOrder(mock) causes error even though I have the right compilation options and the right import. Hence I am not able to get all the tests passing (have disabled for now).

Caused by: java.lang.IllegalArgumentException: Could not parse as Java
	at org.openrewrite.java.internal.template.JavaTemplateParser.lambda$compileTemplate$13(JavaTemplateParser.java:264)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at org.openrewrite.java.internal.template.JavaTemplateParser.compileTemplate(JavaTemplateParser.java:264)
	at org.openrewrite.java.internal.template.JavaTemplateParser.lambda$parseBlockStatements$9(JavaTemplateParser.java:176)
	at org.openrewrite.java.internal.template.JavaTemplateParser.lambda$cacheIfContextFree$14(JavaTemplateParser.java:287)
	at org.openrewrite.java.internal.template.JavaTemplateParser.cache(JavaTemplateParser.java:308)
	at org.openrewrite.java.internal.template.JavaTemplateParser.cacheIfContextFree(JavaTemplateParser.java:287)
	at org.openrewrite.java.internal.template.JavaTemplateParser.parseBlockStatements(JavaTemplateParser.java:171)
	at org.openrewrite.java.internal.template.JavaTemplateJavaExtension$1.lambda$visitBlock$1(JavaTemplateJavaExtension.java:112)
	at org.openrewrite.internal.ListUtils.lambda$flatMap$0(ListUtils.java:261)
	at org.openrewrite.internal.ListUtils.flatMap(ListUtils.java:205)
	at org.openrewrite.internal.ListUtils.flatMap(ListUtils.java:261)
	at org.openrewrite.java.internal.template.JavaTemplateJavaExtension$1.visitBlock(JavaTemplateJavaExtension.java:110)
	at org.openrewrite.java.internal.template.JavaTemplateJavaExtension$1.visitBlock(JavaTemplateJavaExtension.java:55)
	at org.openrewrite.java.tree.J$Block.acceptJava(J.java:838)
	at org.openrewrite.java.tree.J.accept(J.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:250)
	... 58 more

Reply:
I updated branch with main and still seeing the same issue. Strangely it's all working for verifyNoMoreInteractions(mock1, mock2, ...) but doesn't work for InOrder inOrder = inOrder(mock1, mock 2...). If I do just inOrder(mock1, mock2...) it seems to work. This looks like a framework issue, unless I'm missing something here.

Also seeing many of these warnings:
23:18:03.671 [main] WARN org.openrewrite.java.isolated.ReloadableJava17Parser - warning: unknown enum constant Status.STABLE
23:18:03.673 [main] WARN org.openrewrite.java.isolated.ReloadableJava17Parser -   reason: class file for org.apiguardian.api.API$Status not found

Reply:
Ah all good, that parse error went away - thanks, maybe all that was needed was to sync with main! Now just getting some strange formatting issue, just an empy new line between statements in test whenThreeBlocks, hence one test is failing. Not due to having 3 blocks, but due to having a very short InOrder block of 2 lines.